### ISSUE TITLE
Ability to temporarily enable blacklisted sites

### ISSUE THREAD

**Issue by agzam** [1995eaton/chromium-vim#186]
Would be awesome to be able to enable blacklisted website for the session or disable cvim with keyboard.

**Comment by collinpeters** [comment]
:+1:

**Comment by MeanderingCode** [comment]
Agreed.  Something like Pentadactyl's <C-v> to pass the next keypress to Chrome/the web page and <C-z> to enable pass through mode, with <Esc> returning keyhandling to cVim.

Of course, lots of sites like to use <Esc>, as well, and that's never really been elegantly handled in dactyl, but it's still worthwhile.

**Comment by 1995eaton** [comment]
@agzam @collinpeters, is something like @MeanderingCode suggested what you had in mind? I've just added the `passKeys` command, which passes `<N>` keys to the web page.

**Comment by collinpeters** [comment]
That would be useful. You actually solved my primary use case with the fix for #202. So for a site like Gmail I didn't want to fully blacklist it, but rather primarily use it in insert mode as a default, but then still have the ability to go to command mode.
